### PR TITLE
Beta hotfix

### PR DESCRIPTION
**Pull request by ralphiee22**
### Summary

Update theme config to not allow `None` values for scrim opacity.
Check if `brandedFooter` var exists.

### Reviewer guidance

Does sidenav show up now?
Does background image show up now?

----

### Contributor Checklist

PR process:

- [x] PR has the correct target branch and milestone
- [x] PR has 'needs review' or 'work-in-progress' label
- [x] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')
- [ ] If this is an important user-facing change, PR or related issue has a 'changelog' label
- [ ] If this includes an internal dependency change, a link to the diff is provided

Testing:

- [x] Contributor has fully tested the PR manually
- [ ] If there are any front-end changes, before/after screenshots are included
- [ ] Critical user journeys are covered by Gherkin stories
- [ ] Critical and brittle code paths are covered by unit tests

### Reviewer Checklist

- Automated test coverage is satisfactory
- PR is fully functional
- PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- External dependency files were updated if necessary (`yarn` and `pip`)
- Documentation is updated
- Contributor is in AUTHORS.md

### CODE REVIEW

File: kolibri/plugins/user/assets/src/views/SignInPage/index.vue [hunk 1]
```diff
@@ -335,7 +335,7 @@
       backgroundImageStyle() {
         if (this.$theme.signIn.background) {
           const scrimOpacity =
-            this.$theme.signIn.scrimOpacity !== null ? this.$theme.signIn.scrimOpacity : 0.7;
+            this.$theme.signIn.scrimOpacity !== undefined ? this.$theme.signIn.scrimOpacity : 0.7;
```
Comment (indirectlylit) [inline kolibri/plugins/user/assets/src/views/SignInPage/index.vue:338]: could also use `isNaN` here

**Review by indirectlylit — APPROVED** [review]
this looks fine to me. I didn't test it locally, but if it fixed the problem for you let's give it a go. I'll tag another beta once it's merged

File: kolibri/plugins/user/assets/src/views/SignInPage/index.vue (hunk 1 above)
Comment (ralphiee22) [reply kolibri/plugins/user/assets/src/views/SignInPage/index.vue:338]: didn't know about that, but good to know!

Comment (ralphiee22) [reply kolibri/plugins/user/assets/src/views/SignInPage/index.vue:338]: but also I prefer forcing the user to pass in a specific value, hence the opacity `None` checks

**Pull request merged**